src/util/languages/Python.js

```diff
@@ -1,3 +1,14 @@
+import Sk from 'skulpt';
+
+Sk.externalLibraries = {
+    turtle: {
+        path: "/src/util/languages/turtle.js"
+    }
+};
```

Comment:
Hey there, this approach seems pretty intuitive to me. I tested it out and right now `Sk.TurtleGraphics` is properly included in the namespace, but nothing paints, which I assume is intended in the PR's current state.

I've not worked with Skulpt before, but I'm fairly certain that you can directly link its stdlib into the iframe or webpage, which includes `turtle`. This will probably avoid some weird issues with rolling our own, and also prevents bloating our codebase. See their [JSDoc](http://skulpt.org/docs/index.html) for more.

Reply:
I think this is actually unused right now, Turtle should be included by default